_modules/github.js

```diff
@@ -2,8 +2,8 @@
 // #region Imports
 // -----------------------------------------------------------------------------------------
 
+const { echo }    = require("./echo");
 const { Octokit } = require("@octokit/rest");
-const { echo } = require("shelljs");
```

Comment:
I think this would've bombed out attempting to gracefully handle a bombed out command 😂 The `shelljs` version of echo is just a single function, not an object with multiple functions for different messaging.